scripts/install/etc/sysconfig/motr

```diff
@@ -94,6 +94,14 @@ MOTR_M0D_BESEG_SIZE=8589934592
 #MOTR_M0D_BETX_REG_SIZE_MAX=46137344
 #MOTR_M0D_BETX_PAYLOAD_SIZE_MAX=2097152
 
+#BALLOC configuration parameters.
+#
+#BALLOC GROUP COUNT. default is 64
+MOTR_M0D_BALLOC_GROUP_COUNT=64
+
+#BALLOC No of indexes per device, default 1
+MOTR_M0D_BALLOC_INDEX_COUNT=1
```

Comment:
As I commented above, these two parameter can not be changed during the life time of a CORTX deployment?

Reply:
After Mkfs, it has no effect. The changed value will reflect after next time mkfs

Reply:
OK